TypeCobol/Compiler/SqlCodeElements/ISqlCodeElementBuilder.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.Parser.Generated;
+using TypeCobol.Compiler.SqlScanner;
+
+namespace TypeCobol.Compiler.SqlCodeElements
+{
+    /// <summary>
+    /// Interface of a Sql Code Element Builder
+    /// </summary>
+    public interface ISqlCodeElementBuilder
```

Comment:
Not needed. Other builders do not have interface definitions.

Reply:
done in cbf5596